test/test-certs/create-certs.sh

```diff
@@ -41,36 +41,24 @@ openssl req \
     -passin pass:$SSL_SECRET \
     -passout pass:$SSL_SECRET
 
-
-for i in kafka kafka1 kafka2 schema-registry materialized producer postgres certuser materialized-kafka materialized-sr
-do
-
-    if [ "$i" = "materialized-kafka" ] || [ "$i" = "materialized-sr" ];
-    then
-        ca="ca-selective"
-        cn="materialized"
-    else
-        ca="ca"
-        cn="$i"
-    fi
-
+create_cert() {
     # Create key & csr
```

Comment:
Could you add named locals for the positional params? It makes bash functions much easier to follow if you rebind at the start 

eg 
‘’’
local name = “$1”
local ca = “$2”
local cn = “$3”
‘’’